planetiler-benchmarks/src/main/java/com/onthegomap/planetiler/benchmarks/BenchmarkMbtilesRead.java

```diff
@@ -0,0 +1,133 @@
+package com.onthegomap.planetiler.benchmarks;
+
+import com.google.common.base.Stopwatch;
+import com.onthegomap.planetiler.config.Arguments;
+import com.onthegomap.planetiler.geo.TileCoord;
+import com.onthegomap.planetiler.mbtiles.Mbtiles;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.LongSummaryStatistics;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BenchmarkMbtilesRead {
```

Comment:
Overall this read benchmark is looking good, the main thing we care about is throughput of "read operations per second" - could we have the benchmark report that metric? I don't think the per-request stat hurts (unless getting the system time for each request adds overhead) but the main metric I'm interested in is overall read throughput.

Reply:
You are absolutely right, per request stats probably hurts, and I also agree that "read/s" is the better metric. I'll adjust accordingly.

Reply:
addressed here https://github.com/onthegomap/planetiler/pull/219/commits/b8f58f2accc3dfad16d2dd5a86aaa8941c4c4473

Reply:
One thing here however that I cannot wrap my mind around is that after all the refactorings, reading from the compact-db actually seems to be faster. This cannot really be the case but I can't spot an issue in the benchmark.

```
0m0:00:11 INF - working on /.../australia-not-compact.mbtiles
0m0:00:54 INF - readOperationsPerSecondStats: DoubleSummaryStatistics{count=10, sum=1168952.601015, min=86929.648036, average=116895.260102, max=125684.624005}
0m0:00:54 INF - working on /.../australia-compact.mbtiles
0m0:01:35 INF - readOperationsPerSecondStats: DoubleSummaryStatistics{count=10, sum=1241796.798647, min=96633.936167, average=124179.679865, max=130121.770334}
0m0:01:35 INF - diffs
0m0:01:35 INF - "/.../australia-not-compact.mbtiles" vs "/.../australia-compact.mbtiles": avg read operations per second improved by 6.231578386342335%
```

Reply:
Benchmarking sqlite is a bit tricky it seems. I don't quite get it either but currently on my machine it seems to be faster reading from the DB in compact-db mode.